Library/Formula/icetouch.rb

```diff
@@ -0,0 +1,23 @@
+class Icetouch < Formula
+  desc "Implementation of Ice for iOS and OS X targeting Xcode development"
+  homepage "https://zeroc.com"
+  url "https://github.com/zeroc-ice/icetouch.git",
+    :tag => "v3.6.0", :revision => "c39885c589e3c8ecefcfe456ac5c83091b09f927"
+
+  depends_on :macos => :yosemite
+  depends_on "mcpp"
+
+  def install
+    # Unset ICE_HOME as it interferes with the build
+    ENV.delete("ICE_HOME")
+    ENV.delete("USE_BIN_DIST")
+    ENV.O2
+
+    system "make", "install", "OPTIMIZE=yes", "prefix=#{prefix}"
+  end
+
+  test do
+    system "#{lib}/IceTouch/Cpp/bin/slice2cpp", "--version"
+    system "#{lib}/IceTouch/Objc/bin/slice2objc", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!